tools/mason/MasonPublish.chpl

```diff
@@ -38,52 +38,28 @@ proc masonPublish(args: [] string) throws {
       masonPublishHelp();
       exit(0);
     }
-    if args.size == 4 {
-      masonPublishHelp();
-      exit(0);
-    }
     var dry = false;
+    var path = '';
```

Comment:
nit on variable naming: It's easier for me to think of this variable as `registry` or `targetRegistry` rather than `path`.

Reply:
maybe `registryPath` ?